domain/schema/model/sql/0020-unit.sql

```diff
@@ -23,6 +23,11 @@ CREATE TABLE unit (
     resolve_kind_id INT NOT NULL DEFAULT 0,
     password_hash_algorithm_id TEXT,
     password_hash TEXT,
+    track TEXT,
+    risk TEXT NOT NULL,
+    branch TEXT,
```

Comment:
This shouldn't be `track/risk/branch` this is the os channel. We don't know what the form of that is (depending on the OS), so this should just be `channel TEXT NOT NULL`